lib/Sema/DerivedConformanceError.cpp

```diff
@@ -27,12 +27,38 @@
 using namespace swift;
 using namespace swift::objc_translation;
 
-static void deriveBodyBridgedNSError_enum_nsErrorDomain(
-              AbstractFunctionDecl *domainDecl, void *) {
+static void deriveBodyUsingRuntime(AbstractFunctionDecl *domainDecl, void *) {
```

Comment:
Nitpick: derive body of what? (The old name is a hierarchical "derivedBody\<Protocol>\_\<kind>\_\<requirement>".)

Reply:
I know. With all of that *and* the `UsingRuntime`/`UsingObjcTranslation` suffix, the function name was nearing the line length limit on its own. I figured I could drop that information because the file only derives one requirement's body anyway, but I can just use the giant name if you'd prefer.

Reply:
It's more "objcPrintableEnum", but that could work.